webapp/src/Migrations/Version20200425120051.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20200425120051 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return 'Adds deleted boolean field to testcase.';
+    }
+
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE testcase ADD deleted TINYINT(1) DEFAULT \'0\' NOT NULL COMMENT \'Deleted testcases are kept for referential integrity.\', CHANGE probid probid INT UNSIGNED DEFAULT NULL COMMENT \'Corresponding problem ID\', CHANGE orig_input_filename orig_input_filename VARCHAR(255) DEFAULT NULL COMMENT \'Original basename of the input file.\'');
```

Comment:
For readability might be good to break over multiple lines, and (nitpick) also change `CHANGE` into `MODIFY` (dropping duplicate column name, to indicate that the column name didn't change).
Also why did `orig_input_filename` change?

Reply:
That's probably a spurious update generated from the script. I'll fix this and the readability tomorrow.

`CHANGE` vs `MODIFY`: I can change this, but since this is autogenerated, do we really care?

Reply:
The `MODIFY` is really a nitpick: it doesn't change anything but readability. Feel free to leave as is.

Reply:
Done, let's see if gitlab likes it.